hane-syntax/src/parser.rs

```diff
@@ -193,6 +193,31 @@ fn parse_expr_inner(pair: Pair) -> (Span, Expr) {
             debug_assert_rule!(pairs, keyword_end);
             ExprVariant::Match(t, name, pattern, ret, arms)
         }
+        Rule::expr_fix => {
```

Comment:
It would be really nice if some comments of the actual syntax iterated over here were explained